test_scripts/Polices/External_UCS/External_Consent_ON/ATF_Policies_External_Consent_ON_OFF_combine.lua

```diff
@@ -0,0 +1,216 @@
+------------------------------------------------------------------------------------------------------
```

Comment:
@istoimenova, Accepted
Test set is in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1301